### PR TITLE
add FUNC(hasHatch)

### PR DESCRIPTION
**Pull request by commy2**

### CODE REVIEW

File: addons/common/functions/fnc_hasHatch.sqf [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ * Return Value:
+ * Unit has a hatch? <BOOL>
+ *
+ * Public: Yes
+ */
+#include "script_component.hpp"
+
+params ["_unit"];
+
+local _vehicle = vehicle _unit;
```
Comment (BaerMitUmlaut) [inline addons/common/functions/fnc_hasHatch.sqf]: `local`

File: addons/common/functions/fnc_hasHatch.sqf [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+if (_unit == _vehicle) exitWith {false};
+
+local _config = configFile >> "CfgVehicles" >> typeOf _vehicle;
```
Comment (BaerMitUmlaut) [inline addons/common/functions/fnc_hasHatch.sqf]: `local`

File: addons/common/functions/fnc_hasHatch.sqf (hunk 1 above)
Comment (commy2) [inline addons/common/functions/fnc_hasHatch.sqf]: It's the future.

File: addons/common/functions/fnc_hasHatch.sqf [hunk 3]
```diff
@@ -0,0 +1,35 @@
+/*
+ * Author: commy2
+ * Check if unit has a hatch and can turn out.
```
Comment (Glowbal) [inline addons/common/functions/fnc_hasHatch.sqf]: A better description might be: 
> Check if unit in vehicle is in a vehicle position where unit can turn out.

File: addons/common/functions/fnc_hasHatch.sqf (hunk 1 above)
Comment (BaerMitUmlaut) [inline addons/common/functions/fnc_hasHatch.sqf]: Not for long!

File: addons/common/functions/fnc_hasHatch.sqf (hunk 3 above)
Comment (BaerMitUmlaut) [inline addons/common/functions/fnc_hasHatch.sqf]: Might as well call it canTurnOut then? Or does it return something else?

File: addons/common/functions/fnc_hasHatch.sqf (hunk 1 above)
Comment (jonpas) [inline addons/common/functions/fnc_hasHatch.sqf]: There is regex ready to change it, it's still `local` until 1.54 gets rolled and that won't happen for at least 2 more weeks.

**Pull request merged**